site/docs/4.1/components/badge.md

```diff
@@ -72,3 +72,24 @@ Using the contextual `.badge-*` classes on an `<a>` element quickly provide _act
 <a href="#" class="badge badge-{{ color.name }}">{{ color.name | capitalize }}</a>{% endfor %}
 {% endcapture %}
 {% include example.html content=example %}
+
+
+## Dismisable badges
+
+Create "dismissable" badges by adding the `.close` icon within a badge. Note that `<button>` and `<a>` elements are not supported _within_ another `<a>`. No JavaScript is included in Bootstrap to do the dismissing for you, so you'll have to write your own to implement any desired behaviors.
+
+{% capture example %}
+<span class="badge badge-primary">
+  Dismissable badge
+  <button type="button" class="close" aria-label="Dismiss">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</span>
+<a href="#" class="badge badge-secondary">
+  Dismissable link badge
+  <span role="button" class="close" aria-label="Dismiss">
```

Comment:
this...looks pointless. why have a role="button" on a span that's not focusable? would need to do testing, but gut feeling is that no AT will announce/do anything sensible with it when it's part of a link - at best, you'd hear something like "dismissable link badge...dismiss...link". in some cases, the aria-label won't be announced at all, so just "dismissable link badge...link"

Reply:
maybe i'm missing what the whole point of that thing is...why would you want the whole badge to be a link? what's the expected behavior here?